python/tests/test_parquet.py

```diff
@@ -123,6 +154,9 @@ def test_read_array(tmp_path, npartitions=2, glob_string="/*"):
     assert arr.dtype == arr_from_tbl.dtype
     assert cn.array_equal(arr_from_tbl, arr)
 
+    assert arr.dtype == arr_from_tbl.dtype
+    assert np.array_equal(arr_from_tbl, arr)
+
```

Comment:
```suggestion
```
Or was there a reason, seems just a repeat of the above.

(Unless we want to write this with only numpy and avoid cupynumeric, but not sure if legate supports that for its logical array.)